src/DynamoCoreWpf/UI/Prompts/NodeHelpPrompt.xaml.cs

```diff
@@ -19,11 +22,21 @@ public NodeHelpPrompt(NodeModel node)
             this.DataContext = node;
             this.WindowStartupLocation = WindowStartupLocation.CenterOwner;
             InitializeComponent();
+            
+            if (node.IsCustomFunction)
+            {
+                // Hide the dictionary link if the node is a custom node
+                DynamoDictionaryHeight.Height = new GridLength(0);
+            }
         }
 
         private void OpenDynamoDictionary(object sender, MouseButtonEventArgs e)
         {
-            Process.Start(new ProcessStartInfo("explorer.exe", ((TextBlock)sender).Tag.ToString()));
+            var dynView = this.Owner.DataContext as DynamoViewModel;
+            var node = this.DataContext as NodeModel;
+            node.DictionaryLink = node.GetDictionaryLinkFromLibrary(dynView.Model.LibraryServices);
```

Comment:
Can we make `DictionaryLink` have a private setter and make a call to `node.GetDictionaryLinkFromLibrary` in its getter?